help/de/Help_ch_Account-Actions.xml

```diff
@@ -2456,17 +2388,551 @@ and update the appendix. -->
+        <title>Assistent für den Export der Kontenhierarchie im CSV-Format</title><screenshot id="export-accounts-wizard">
+          <mediaobject>
+            <imageobject role="html">
+              <imagedata fileref="figures/Help_Export_Acct_CSV.png" format="PNG" srccredit="Christian Wehling" width="510px" />
+            </imageobject>
+
+            <imageobject role="fo">
+              <imagedata fileref="figures/Help_Export_Acct_CSV.png" format="PNG" srccredit="Christian Wehling" />
+            </imageobject>
+          </mediaobject>
+        </screenshot>
+      </figure>
+
+      <para>In jedem Fenster kommen diese Navigationsschaltflächen zur Anwendung:
+      </para>
+
+      <variablelist spacing="compact">
+        <title>Dialogführung</title>
+
+        <varlistentry>
+          <term><guilabel>Abbrechen</guilabel></term>
+
+          <listitem>
+            <para>Beendet den Assistenten, ohne die Kontenhierarchie zu exportieren.
+            </para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><guilabel>Zurück</guilabel></term>
+
+          <listitem>
+            <para>Geht zum vorherigen Schritt.
+            </para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><guilabel>Weiter</guilabel></term>
+
+          <listitem>
+            <para>Geht zum nächsten Schritt.
+            </para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+
+      <note>
+        <para>Die vorgenannten Schaltflächen werden erst dann aktiv, wenn die Navigation zum Zielschritt vom
+          aktuellen Schritt aus gültig ist.
+        </para>
+      </note>
+
+      <variablelist spacing="compact">
+        <title>Schritte des Exportdialogs</title>
+
+        <varlistentry id="acct-export-csv-settings">
+          <term><guilabel>Export-Einstellungen</guilabel></term>
+
+          <listitem>
+            <para>In diesem Schritt können Sie verschiedene Vorgaben für die Formatierung der zu exportierenden
+              Konten machen.
+            </para>
+
+            <variablelist spacing="compact">
+              <varlistentry>
+                <term><guilabel>Optionen</guilabel></term>
+
+                <listitem>
+                  <para>Dieses Kontrollkästchen kann zusätzlich zu dem <guilabel>Trennzeichen</guilabel> verwendet werden:
+                  </para>
+
+                  <variablelist spacing="compact">
+                    <varlistentry>
+                      <term><guilabel>Anführungszeichen verwenden</guilabel></term>
+
+                      <listitem>
+                        <para>Umgibt jedes Datenfeld mit Anführungszeichen.
+                        </para>
+                      </listitem>
+                    </varlistentry>
+                  </variablelist>
+                </listitem>
+              </varlistentry>
+
+              <varlistentry>
+                <term><guilabel>Trennzeichen</guilabel></term>
```

Comment:
Können wir das mit den anderen Vorkommen zusammenfassen? Ggf. eigene Seite impex-delimiter or impex-common-options? Sonst müßten wir die ganzen Anmerkungen wiederholen.
Oder eigene Datei, die jeweils eingebunden wird.
BP

Reply:
Die sind zum einen nur in 3 von 4 Punkten richtig übereinstimmend, und zum anderen bekomme ich das innerhalb der `<procedure>` und den `<step>`s nicht hin.